dnsdbq.c

```diff
@@ -2785,27 +2866,21 @@ timecmp(u_long a, u_long b) {
 	return (0);
 }
 
-/* time_print -- format one (possibly relative) timestamp.
+/* time_str -- format one (possibly relative) timestamp (returns static string)
```

Comment:
While the callers of this revised function look much cleaner, I don't like the risk of a caller not realizing it is static.  Is there a function naming convention to indicate *not* reentrant etc.?